src/cpp/tests/proxyenginetest.cpp

```diff
@@ -740,7 +746,9 @@ private slots:
 	{
 		reset();
 
-		QSignalSpy spy(engine->statsManager(), SIGNAL(reported(const QList<StatsPacket> &)));
+        boost::signals2::scoped_connection reportConnection = engine->statsManager()->reported.connect(
+            boost::bind(&ProxyEngineTest::appendTrackedPackets, this, boost::placeholders::_1)
+        );
```

Comment:
indentation (this and all the other instances)

Reply:
😢